src/entities/SaveHistory.ts

```diff
@@ -0,0 +1,83 @@
+import * as env from "@utils/config.constant";
+import fs from 'fs';
+
+interface HistoryEntry {
+  Epoch: number;
+  
+}
+
+export class SaveHistory {
```

Comment:
Naming the class `History` might make more sense instead of a `SaveHistory`. Then all of the public methods like `appendToHistory`, `write_history_json`, `maintain_history_length`, etc. can just be named `append`, `writeJson`, `maintainLength`, etc.